src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.impl.response.OrderResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+    private final OrderService orderService;
+    private final OrderResponseMapper orderResponseMapper;
+
+    @Autowired
+    public OrderController(UserService userService,
+                           ShoppingCartService shoppingCartService,
+                           OrderService orderService,
+                           OrderResponseMapper orderResponseMapper) {
+        this.userService = userService;
+        this.shoppingCartService = shoppingCartService;
+        this.orderService = orderService;
+        this.orderResponseMapper = orderResponseMapper;
+    }
+
+    @PostMapping("/complete")
+    public OrderResponseDto completeOrder(@PathVariable Long userId) {
```

Comment:
Please recheck all controllers, I believe you wanted to use RequestParam instead of PathVariable